pym/bob/cmds/jenkins.py

```diff
@@ -375,6 +375,8 @@ def dumpStepAuditGen(self, step):
             "-D", "jenkins-node", '"$NODE_NAME"',
             "-D", "jenkins-build-url", '"$BUILD_URL"'
         ]
+        for (var, val) in step.getPackage().getMetaEnv().items():
+            cmd.extend(["--metaEnv", var, val])
```

Comment:
`val` must be quoted. Maybe using a short option (`-E`?) could save some space...